packages/overlay/test/interactions.common.js

```diff
@@ -293,5 +303,27 @@ describe('interactions', () => {
         expect(overlay.opened).to.be.true;
       });
     });
+
+    describe('mousedown on content', () => {
+      it('should prevent default on content mousedown', () => {
+        const div = overlay.querySelector('div');
+        const event = makeMouseEvent('mousedown', middleOfNode(div), div);
+        expect(event.defaultPrevented).to.be.true;
```

Comment:
I was trying to understand why it is necessary to cancel the event to prevent the `focusout` from occurring. This test case kind of focuses on a technical detail without explaining why it's necessary to do so.

Reply:
Good point. I will try to use `sendMouse` to check that there is no `focusout` event.

Reply:
Added a test.